crates/sdk/Cargo.toml

```diff
@@ -122,6 +122,16 @@ cuda = [
     "openvm-rv32im-circuit/cuda",
 ]
 
+# Legacy compatibility features for backwards compatibility with v1.2/v1.3
+legacy = ["legacy-evm-verifier", "legacy-mem-align"]
```

Comment:
not sure on standard syntax, but legacy is ambiguous, should we say `legacy-v1-3` here and all other uses of `legacy`?

Reply:
solved